### PR TITLE
Preliminary automation for multi window ipad

### PR DESCRIPTION
**Pull request by dragosb01**
## :scroll: Tickets
https://mozilla-hub.atlassian.net/browse/MTE-2870

## :bulb: Description
This PR propose to add preliminary functionality for future tests on multi window.
The tests added are just for demo and are not based on test rail scenarios. These will be updated/removed.

### CODE REVIEW

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/MultiWindowTests.swift [hunk 1]
```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+class MultiWindowTests: IpadOnlyTestCase {
```
Comment (isabelrios) [inline firefox-ios/firefox-ios-tests/Tests/XCUITests/MultiWindowTests.swift:7]: Nit: add extra line

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/MultiWindowTests.swift [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    private func splitViewFromHomeScreen() {
+        mozWaitForElementToExist(dotMenu)
+        dotMenu.tap()
+        mozWaitForElementToExist(splitView)
+        splitView.tap()
+        springboard.icons.elementContainingText("split view with Fennec").tap()
+    }
+
+    // windowsNumber param - number of tab windows to open from switcher
```
Comment (isabelrios) [inline firefox-ios/firefox-ios-tests/Tests/XCUITests/MultiWindowTests.swift:53]: Nit: start with capital W

**Comment by isabelrios** [comment]
Just to confirm, this will be running daily as part of the Full Functional test suite

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/MultiWindowTests.swift (hunk 2 above)
Comment (dragosb01) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/MultiWindowTests.swift]: Updated

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/MultiWindowTests.swift (hunk 1 above)
Comment (dragosb01) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/MultiWindowTests.swift:8]: Updated

**Review by isabelrios — APPROVED** [review]

**Pull request merged**